src/why/Why.story.mdx

```diff
@@ -0,0 +1,36 @@
+import { action } from '@storybook/addon-actions'
+import { Meta, Preview, Props, Story } from '@storybook/addon-docs/blocks'
+
+import { FocusVisibleProvider } from '../_utils/focusVisibleProvider'
+import { Why } from './index'
+
+<Meta title="Widgets|Why" />
+
+# **Why**
+
+<Preview>
+  <Story name="Default">
+    <div>
```

Comment:
why do we need this div here?

Reply:
Copy/Pasted what we had the previous story. I'll check